spec/rubocop/rspec/r_spec_config_spec.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+RSpec.describe RuboCop::RSpec::RSpecConfig, :config do
+  include RuboCop::RSpec::Language::NodePattern
+
+  let(:cop_class) { RuboCop::Cop::Base }
+  let(:source) do
+    <<~RUBY
+      epic 'great achievements or events is narrated in elevated style' do
+        ballad 'slay Minotaur' do
+          # ...
+        end
+      end
+    RUBY
+  end
+  let(:ast) { RuboCop::ProcessedSource.new(source, RUBY_VERSION.to_f).ast }
+
+  context 'with the default config' do
```

Comment:
@bquorning The test for this whole change.

A real cop spec: https://github.com/rubocop-hq/rubocop-rspec/pull/1007/commits/0cf00f4c26e38e5658039c6ac41d5d01f6b2d922#diff-fb6cde18a3a09cdce77660231314ae226a9fd3605928279010b8dbd3558e518eR271